reports-scheduler/src/main/java/com/amazon/opendistroforelasticsearch/reportsscheduler/metrics/BasicCounter.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
np: copyright should be 2020 or 2021

Reply:
Good catch! Changed to 2020 from 2019 at all places.